sphinx_automodapi/tests/test_automodapi.py

```diff
@@ -1,52 +1,39 @@
 # -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-import pytest
+import sys
+from copy import copy
 
-from ..utils import iteritems
-from . import cython_testpackage
+import pytest
+from docutils.parsers.rst import directives, roles
 
-pytest.importorskip('sphinx')  # skips these tests if sphinx not present
+from . import cython_testpackage  # noqa
+from .helpers import run_sphinx_in_tmpdir
 
+if sys.version_info[0] == 2:
+    from io import open as io_open
+else:
+    io_open = open
 
-class FakeConfig(object):
-    """
-    Mocks up a sphinx configuration setting construct for automodapi tests
-    """
-    def __init__(self, **kwargs):
-        for k, v in iteritems(kwargs):
-            setattr(self, k, v)
-
-
-class FakeApp(object):
-    """
-    Mocks up a `sphinx.application.Application` object for automodapi tests
-    """
+pytest.importorskip('sphinx')  # skips these tests if sphinx not present
```

Comment:
Is this ever possible for this package?

Reply:
I think this is actually left over from when it was in astropy-helpers. Will remove.

Reply:
Fixed